File: app/src/main/java/com/ne1c/gitteroid/api/responses/JoinRoomResponse.java
package com.ne1c.gitteroid.api.responses;

import android.annotation.SuppressLint;

import com.google.gson.annotations.SerializedName;
import com.ne1c.gitteroid.models.data.RoomModel;

@SuppressLint("ParcelCreator")
public class JoinRoomResponse extends RoomModel {
    @SerializedName("error")
    private String mError;

    public String getError() {
        return mError;
    }
}

File: app/src/main/kotlin/com/ne1c/gitteroid/api/responses/JoinRoomResponse.kt
package com.ne1c.gitteroid.api.responses

import android.annotation.SuppressLint
import com.google.gson.annotations.SerializedName
import com.ne1c.gitteroid.models.data.RoomModel

@SuppressLint("ParcelCreator")
class JoinRoomResponse : RoomModel(){
    @SerializedName("error")
    val error: String? = null
}
